Source/Dafny/Triggers/QuantifiersCollection.cs

```diff
@@ -299,32 +307,42 @@ internal class QuantifierGroup {
         }
 
         AddTriggersToMessage("Selected triggers:", q.Candidates, msg, indent);
+        if (q.Candidates.Count > 0) {
+          SetRelatedLocationMessage("Trigger position:");
+        }
         AddTriggersToMessage("Rejected triggers:", q.RejectedCandidates, msg, indent, true);
+        if (q.RejectedCandidates.Count > 0 && q.Candidates.Count == 0) {
+          SetRelatedLocationMessage("Rejected trigger position:");
```

Comment:
There's more than one rejected trigger typically, so adding a link to just one of them is a bit weird